packages/ng-primitives/schematics/radio/generator.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright © 2024 Angular Primitives.
+ * https://github.com/ng-primitives/ng-primitives
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { formatFiles, generateFiles, names, Tree } from '@nx/devkit';
+import * as path from 'path';
+import { RadioGeneratorSchema } from './schema';
+
+const radioItemStyles = `
+:host {}
+
+:host[data-hover="true"] {}
+
+:host[data-focus-visible="true"] {}
+
+:host[data-press="true"] {}
+
+:host[data-checked="true"] {}
+
+[ngpRadioIndicator] {}
+
+:host[data-checked="true"] [ngpRadioIndicator] {}
+
+.indicator-dot {}
+
+:host[data-checked="true"] .indicator-dot {}
+
+p {}
+
+:host[data-checked="true"] p {}`;
```

Comment:
I think we can remove a few of these rules, we don't know that there would be a `p` tag or an `.indicator-dot` etc.., so we can remove these and keep only the things that are a "given"

```suggestion
:host {}

:host[data-hover="true"] {}

:host[data-focus-visible="true"] {}

:host[data-press="true"] {}

:host[data-checked="true"] {}

[ngpRadioIndicator] {}

:host[data-checked="true"] [ngpRadioIndicator] {}
`;
```